### PR TITLE
Add preliminary date and locations

### PR DESCRIPTION
**Pull request by EuiYeonJang**
Add the date of the prelimaries and add a list of locations. Added locations for NL only at the moment, perhaps we can later include information about universities that participate in general for completeness.

### CODE REVIEW

**Comment by EuiYeonJang** [comment]
@shanemin or @mvr320 how do you fix the issue that the grammar checker thinks the bold syntax is a multiplication?

File: content/registration.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ weight: 2
 
 The registration will open after the preliminaries have taken place. 
 
-The slots per institution for BAPC21 are limited. As there tends to be more teams than slots available, your institution can hold preliminary contests to determine which teams qualify for BAPC21. The preliminaries are expected to be held in the summer of 2021. 
+## Preliminaries
+
+The slots per institution for BAPC21 are limited. As there tends to be more teams than slots available, your institution can hold preliminary contests to determine which teams qualify for BAPC21. The preliminaries will hold on **Saturday, 9 October**.
```
Comment (mvr320) [inline content/registration.md:13]:
```suggestion
The slots per institution for BAPC21 are limited. As there tends to be more teams than slots available, your institution can hold preliminary contests to determine which teams qualify for BAPC21. The preliminaries should hold on **Saturday, 9 October**.
```

**Comment by mvr320** [comment]
> 
> 
> @shanemin or @mvr320 how do you fix the issue that the grammar checker thinks the bold syntax is a multiplication?

Not making it bold. I think the focus on the date is not important as we're not in charge of sites picking the right date. We might have an opinion but most of this discussion takes place on Slack.

File: content/registration.md (hunk 1 above)
Comment (EuiYeonJang) [reply content/registration.md:13]: Will you be okay with "planned for Saturday, 9 October"?

Comment (mvr320) [reply content/registration.md]: Yes I think thats fine.

Or not mention it here as I prefer to have 1 schedule page. Those mentions can be misleading if we forget to update them. The people registering will know of such things. But if you want to keep the date there than the formulation is fine by me.

**Review by mvr320 — APPROVED** [review]

**Pull request merged**